boom-boot.spec

```diff
@@ -154,6 +154,9 @@ rm doc/conf.py
 
 
 %changelog
+* Fri Dev 06 2024 Bryn M. Reeves <bmr@redhat.com> - 1.6.5-3
```

Comment:
Oops... 
```suggestion
* Fri Dec 06 2024 Bryn M. Reeves <bmr@redhat.com> - 1.6.5-3
```

Reply:
Doh! Good catch, thanks for pointing that out. This was just a scratch branch to trigger a new packit build but I'll get that fixed up (I'm a bit surprised that the RPM builds didn't complain about an invalid %changelog date...).